container-engine-lib/lib/backend_interface/objects/configs/remote_backend_config.go

```diff
@@ -0,0 +1,58 @@
+package configs
+
+import (
+	"encoding/json"
+	"fmt"
+
+	contexts_config_store_generated_api "github.com/kurtosis-tech/kurtosis/contexts-config-store/api/golang/generated"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+// TODO: Move to the backend interface dir to be more generic
+const (
+	urlScheme = "tcp"
+)
+
+var (
+	NoRemoteBackendConfig *KurtosisRemoteBackendConfig = nil
+)
+
+type KurtosisRemoteBackendConfig struct {
```

Comment:
The remote backend config is backend agnostic with a generic endpoint.  It is currently only used with the remote docker backend.

Reply:
in code